ls1intum/Artemis
# `Text Exercises`: Fix Missing Route to Cluster Statistics in Exams

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

*`The rest of checkboxes are irrelevant to this PR, as the changes are minimal`

### Motivation and Context


Link to -> Cluster Statistics Page in Text Exercise detail page was missing.

### Description

Adds route to link to Cluster Statistics page for Text Exercises

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exam Text Exercise with Automatic Assessment enabled

1. Log in to Artemis
2. Navigate to Course Administration
3. Find an exam with a text exercise.. or just access directly one of these instead (Test user 15): 
   - [exam1 > text-exercise](https://artemistest.ase.in.tum.de/course-management/482/exams/1434/exercise-groups/2653/text-exercises/11076)
   - [exam2 > text-exercise](https://artemistest.ase.in.tum.de/course-management/482/exams/1433/exercise-groups/2652/text-exercises/11074)
   - [exam3 > text-exercise](https://artemistest.ase.in.tum.de/course-management/482/exams/1432/exercise-groups/2651/text-exercises/11068)
   - [exam4 > text-exercise](https://artemistest.ase.in.tum.de/course-management/482/exams/1431/exercise-groups/2650/text-exercises/11067)
4. Try to click on "Cluster Statistics"
![image](https://user-images.githubusercontent.com/18335489/137563900-694d0f4f-1d6a-4dbf-bb51-547fd0070186.png)

5. The Cluster Statistics page should show.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots


_This section is irrelevant to the PR_
